ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/handler/TestAuditQueryHandler.java

```diff
@@ -96,13 +96,13 @@
     public static void beforeAll()
     {
         ClientInitializer.beforeClass();
-        oldPartitionerToRestore = DatabaseDescriptor.setPartitionerUnsafe(Mockito.mock(IPartitioner.class));
+        //oldPartitionerToRestore = DatabaseDescriptor.setPartitionerUnsafe(Mockito.mock(IPartitioner.class));
     }
 
     @AfterClass
     public static void afterAll()
     {
-        DatabaseDescriptor.setPartitionerUnsafe(oldPartitionerToRestore);
+        //DatabaseDescriptor.setPartitionerUnsafe(oldPartitionerToRestore);
```

Comment:
Remove

Reply:
done (again?) don't know if something strange happened with the merge or if I missed this the first time...